search-commons/src/main/java/no/unit/nva/search/common/QueryFilter.java

```diff
@@ -43,6 +47,15 @@ public int size() {
     return filters.size();
   }
 
+  public boolean hasTermsQuery(String queryName, String fieldName, Object... values) {
+    var query = this.filters.get(queryName);
+    if (nonNull(query) && query instanceof TermsQueryBuilder termsQueryBuilder) {
+      return termsQueryBuilder.fieldName().equals(fieldName)
+          && new HashSet<>(termsQueryBuilder.values()).containsAll(List.of(values));
+    }
```

Comment:
This returns false if I enter nonsense like "heart=emoji"?

Reply:
These are term-based queries applied programmatically as static filters. The method is only present for tests to be able to inspect and assert on presence of such filters.